### PR TITLE
Fix broken web client build for vega plugin

### PR DESCRIPTION
**Pull request by zachmullen**
An upstream of the vega plugin (datalib) changed to require
loading of a json file. This change extends our loaders so that
the JSON loading rule also applies under node_modules, which
seems pretty harmless.

This is blocking our CI for the moment, could any @girder/developers give it a quick QA so we can get builds rolling again?

### CODE REVIEW

File: grunt_tasks/webpack.config.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ module.exports = {
             // JSON files
             {
                 test: /\.json$/,
-                include: loaderPaths,
+                include: [...loaderPaths, ...loaderPathsNodeModules],
```
Comment (jbeezley) [inline grunt_tasks/webpack.config.js]: Did we ever decide on a minimum node version for development?  This requires v4, I think.

Comment (zachmullen) [inline grunt_tasks/webpack.config.js]: v4 is now what our docs prescribe for installation. (We are actually already using ES2015 features in our build code besides this).

Comment (jbeezley) [inline grunt_tasks/webpack.config.js]: Okay, LGTM then.

**Comment by zachmullen** [comment]
thanks @jbeezley !

**Pull request merged**